shadow-dom/reference-target/tentative/commandfor.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE HTML>
```

Comment:
Should this file be commandfor.tentative.html seeing as commandfor isn't in the HTML spec yet?